dapr/clients/grpc/_response.py

```diff
@@ -933,6 +933,64 @@ def extended_metadata(self) -> Dict[str, str]:
         return self._extended_metadata
 
 
+# RRL TODO: Add properties and fix init
+class GetWorkflowResponse():
+    '''The response of get_workflow operation.'''
+
+    def __init__(
+        self,
+        client: DaprGrpcClient,
+        instance_id: str,
+        workflow_name: str,
+        created_at: str,
+        last_updated_at: str,
+        runtime_status: str,
+        properties: Dict[str, str] = {},
+        headers: MetadataTuple = (),
+    ):
+        """Initializes a GetWorkflowResponse.
+
+        Args:
+            client (DaprClient): a reference to the dapr client used for the GetWorkflow request.
+            instance_id (str): the instance ID assocated with this response.
+            workflow_name (str): the name of the workflow that was started.
+            created_at (str): the time at which the workflow started executing.
+            last_updated_at (str): the time at which the workflow was last updated.
+            runtime_status (str): the current runtime status of the workflow.
+            properties (Dict[str, str]): properties sent as a reponse by the workflow.
+            headers (Tuple, optional): the headers from Dapr gRPC response.
+        """
+        super().__init__(headers)
```

Comment:
You cannot call super because you are not extending a parent class here. Did you mean to use DaprResponse?

Reply:
This is probably the root cause of the Type check error by the way (`tox -e type`)